lib/puppet/indirector/file_server.rb

```diff
@@ -15,7 +15,15 @@ def authorized?(request)
 
     # If we're not serving this mount, then access is denied.
     return false unless mount
-    mount.allowed?(request.node, request.ip)
+
+    # If there are no auth directives or there is an 'allow *' directive, then
+    # access is allowed.
+    if mount.empty? || mount.globalallow?
```

Comment:
Defaulting to deny and requiring an `allow *` for each mountpoint seems like the most secure option, but I'm not a huge fan of it. We're explicitly removing authorization rules from `fileserver.conf`, and removing authorization will mean that at some point we're going to change to default allow.

Do we have long term plans to replace the fileserver with another implementation or service? If so I'm okay-ish with requiring `allow *` on each mountpoint as a stopgap until we replace this service. However if we don't have such plans I dislike leaving this technical debt lying as cruft.

Do we have a feel for how widely custom fileserver.conf mountpoints are used?